src/orchard/stacktrace/parser.clj

```diff
@@ -0,0 +1,42 @@
+(ns orchard.stacktrace.parser
+  (:require [orchard.misc :refer [safe-read-edn]]
+            [orchard.stacktrace.parser.aviso :as aviso]
+            [orchard.stacktrace.parser.clojure.repl :as clojure.repl]
+            [orchard.stacktrace.parser.clojure.stacktrace :as clojure.stacktrace]
+            [orchard.stacktrace.parser.clojure.tagged-literal :as clojure.tagged-literal]
+            [orchard.stacktrace.parser.clojure.throwable :as clojure.throwable]
+            [orchard.stacktrace.parser.java :as java]))
+
+(def default-parsers
+  "The default stacktrace parsers."
+  [clojure.throwable/parse-stacktrace
+   clojure.tagged-literal/parse-stacktrace
+   clojure.stacktrace/parse-stacktrace
+   java/parse-stacktrace
+   clojure.repl/parse-stacktrace
+   aviso/parse-stacktrace])
+
+(def default-input-transformations
+  "The default input transformations."
+  [identity safe-read-edn (comp safe-read-edn safe-read-edn)])
```

Comment:
Composing a function with itself looks a bit weird, but I guess you know what you're doing. :D

Reply:
I'm actually thinking of removing this. `safe-read-edn` is useful if an exception is buried inside a printed string. The `(comp safe-read-edn safe-read-edn)` just does this twice, but I think it is a rare use case.

Reply:
WDYT?